packages/keycloak-user-management/src/components/UserList/TableActions/tests/index.test.tsx

```diff
@@ -4,6 +4,13 @@ import { BrowserRouter } from 'react-router-dom';
 import { TableActions } from '..';
 import * as fixtures from './fixtures';
 import { shallow, mount } from 'enzyme';
+import { fireEvent } from '@testing-library/react';
+import * as reactQuery from 'react-query';
+import * as utils from '../utils';
+import flushPromises from 'flush-promises';
+import { act } from 'react-dom/test-utils';
```

Comment:
act is re-exported by RTL. Recommended to use that https://testing-library.com/docs/react-testing-library/api/#act